infra/base-images/base-builder/Dockerfile

```diff
@@ -50,7 +50,9 @@ RUN cd /tmp/ && \
     cd Python-$PYTHON_VERSION && \
     ./configure --enable-optimizations && \
     make -j install && \
-    rm -r /tmp/Python-$PYTHON_VERSION.tar.xz /tmp/Python-$PYTHON_VERSION
+    cd .. && \
+    rm -r /tmp/Python-$PYTHON_VERSION.tar.xz /tmp/Python-$PYTHON_VERSION && \
+    apt-get remove -y libbz2-dev  # https://github.com/google/oss-fuzz/issues/3888
```

Comment:
nit: install this in the same RUN as well (i.e. move apt-get install libbz2-dev to this RUN), to avoid creating a layer with it.